test_submissions/civic/create_civic_therapy_submissions.ipynb

```diff
@@ -0,0 +1,535 @@
+    "  \"\"\"Get assertion type for clinical impact\n",
+    "\n",
+    "  :param study: GKS study\n",
+    "  :param study_id: ID for ``study``\n",
+    "  :return: Assertion type for clinical impact data if all components found. Otherwise,\n",
+    "    ``None``\n",
+    "  \"\"\"\n",
+    "  assertion_type_for_clinical_impact = None\n",
+    "  if study[\"predicate\"] == \"predictsSensitivityTo\":\n",
+    "    assertion_type_for_clinical_impact = \"therapeutic: sensitivity/response\"\n",
+    "  elif study[\"predicate\"] == \"predictsResistanceTo\":\n",
+    "    assertion_type_for_clinical_impact = \"therapeutic: resistance\"\n",
+    "  else:\n",
+    "    logging.debug(\"%s: no assertion_type_for_clinical_impact\", study_id)\n",
+    "\n",
+    "  return assertion_type_for_clinical_impact\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def _get_variant_set(study: dict, study_id: str) -> dict:\n",
+    "  \"\"\"Get variant set data\n",
+    "\n",
+    "  :param study: GKS study\n",
+    "  :param study_id: ID for ``study``\n",
+    "  :return: Variant set data if all components found. Otherwise, empty dict\n",
+    "  \"\"\"\n",
+    "  hgvs_c = None\n",
+    "  for expr in study[\"variant\"][\"definingContext\"].get(\"expressions\") or []:\n",
+    "    if expr[\"syntax\"] == \"hgvs.c\":\n",
+    "      hgvs_c = expr[\"value\"]\n",
+    "  if not hgvs_c:\n",
+    "    logging.debug(\"%s: no hgvs_c\", study_id)\n",
+    "    return {}\n",
+    "\n",
+    "  ncbi_gene = None\n",
+    "  for mapping in study[\"qualifiers\"][\"geneContext\"][\"mappings\"]:\n",
+    "    source, code = mapping[\"coding\"][\"code\"].split(\":\")\n",
+    "    if source == \"ncbigene\":\n",
+    "      ncbi_gene = int(code)\n",
+    "  if not ncbi_gene:\n",
+    "    logging.debug(\"%s: no ncbi_gene\", study_id)\n",
+    "    return {}\n",
+    "\n",
+    "  return {\n",
+    "    \"variant\": [{\n",
+    "      \"hgvs\": hgvs_c,\n",
+    "      \"gene\": [{\"id\": ncbi_gene}]\n",
+    "    }]\n",
+    "  }"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_clinvar_sub(study: dict) -> dict | None:\n",
+    "  \"\"\"Create a ClinVar clinical impact submission for CIViC data\n",
+    "\n",
+    "  :param study: CIViC study in GKS format\n",
+    "  :return: ClinVar impact submission if able to create from study data. Otherwise,\n",
+    "    ``None``\n",
+    "  \"\"\"\n",
+    "  study_id = study[\"id\"]\n",
+    "  clinvar_impact_sub = {\n",
+    "    \"recordStatus\": \"novel\",\n",
+    "    \"localID\": study[\"variant\"][\"id\"],\n",
+    "    \"localKey\": study_id,\n",
+    "    \"observedIn\": [{\n",
+    "      \"alleleOrigin\": \"somatic\",\n",
+    "      \"affectedStatus\": \"yes\",\n",
```

Comment:
I'm hardcoding this. I think I should actually be using the `Direction` in the evidence item to determine this? @ahwagner 

```json
"affectedStatus": {
          "type": "string",
          "description": "Indicates whether or not the individual(s) in each observation were affected by the condition for the interpretation",
          "enum": [
            "yes",
            "no",
            "unknown",
            "not provided",
            "not applicable"
          ]
```